### PR TITLE
(appengine) dynamic branch from webhook

### PR DESCRIPTION
**Pull request by danielpeach**
@lwander or @duftler please review. Depends on https://github.com/spinnaker/orca/pull/1112

screen shot 2017-01-10 at 10 34 40 am

### CODE REVIEW

File: app/scripts/modules/appengine/serverGroup/configure/wizard/basicSettings.html [hunk 1]
```diff
@@ -113,15 +138,7 @@ <h5 class="text-center">
...
-                There is already a server group in this cluster. Do you want to clone it?
-              </p>
-              <p>
-                Cloning copies the entire configuration from the selected server group, allowing you
-                to modify whichever fields (e.g. image) you need to change in the new server group.
-              </p>
-              <p>
-                To clone a server group, select "Clone" from the "Server Group Actions" menu in the details view of the
-                server group.
+                There is already a server group in this cluster.
```
Comment (lwander) [inline app/scripts/modules/appengine/serverGroup/configure/wizard/basicSettings.html]: Why remove that help-text?

Comment (danielpeach) [inline app/scripts/modules/appengine/serverGroup/configure/wizard/basicSettings.html]: I don't think there will be a clone operation for App Engine, which is why I removed these lines. But I think that would be best done in a different PR.
